src/webdriver.cr

```diff
@@ -84,10 +84,10 @@ module Selenium
 
     private def failure(response)
       if response.headers["Content-Type"].starts_with?("application/json")
-        body = JSON.parse(response.body).raw.as(Hash)
-        status = body["status"].as(Int)
-        value = body["value"].as(Hash)
-        raise Selenium.error_class(status).new(value["message"].as(String))
+        body = JSON.parse(response.body)
+        status = body["status"].as_i
+        value = body["value"]
+        raise Selenium.error_class(status).new(body["value"]["message"].to_s)
```

Comment:
You don't need `value = body["value"]` anymore and this it should be `body["value"]["message"].as_s`.

That's part of the reason I don't like `JSON::Type`, it introduces confusing namings: `.as_x` vs `.to_x`.

Reply:
Yeah that is confusing. I'm not sure I understand the difference :S

Reply:
> Yeah that is confusing. I'm not sure I understand the difference :S

- `as_x`: "type cast", type is already `x`
- `to_x`: "type covert", type is not `x` (commonly)

Reply:
That makes sense. Thank you :)